style.css

```diff
@@ -0,0 +1,164 @@
+.personal input {
+  display: table-cell;
+  width: 80%;
+  padding: 3%;
+  margin-right: 45px;
+}
+td {
+  margin-right: 45px;
+}
+label {
+  display: table-cell;
+  size: 300px;
+}
+textarea {
+  display: table-cell;
+}
+
+body {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+.row2 {
+  display: grid;
+  grid-template-columns: 50% 50%;
+}
+.box {
+  max-width: 90%;
+}
+.box2 {
+  max-width: 90%;
+}
+.box3 {
+  display: inline-block;
+}
+.per-details {
+  margin-bottom: 20px;
+}
+
+textarea {
+  width: 95%;
+  padding: 2%;
```

Comment:
Please always try px insisted of %.

Reply:
tried pixels but % is working much better